shell.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
```

Comment:
should we add `set -e` here?

Reply:
@guewen I do not see the interest. 
We may add a trapp to hide error to customer and log something useful for us